### DISCUSSION TITLE
Micro's CNE TimerMod (v1.3) [BIG CUSTOMIZATION]

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#417] [Show and tell]
## Features:
- Cool Timer
- Cool colors for the timer
- Visual settings for the timer
- Downscroll support
- Psych Style Timer option
- Visual settings for psych timer
- Fuck you I'm not adding the Kade engine timer
- Submenu in the credits (it's pretty small)
- More to come hopefully (I'm taking suggestions)

## Current Download Link:
https://drive.google.com/file/d/1l3-hkogwzd-g1NnIpfE3-fhTnTyoRDVb/view?usp=sharing

## Installation Instructions:
- The mod has been moved to an addons folder, as the previous integration SUCKED.
- Drag the mod folder into your addons folder
- Badabing badaboom

(Recommendations: my favorite setting is Micro Style with Dad Color)

**Comment by fluffabun15** [comment]
really cool! yk u can also post your scripts and stuff in the codename discord?

**Reply by THEkatinamicrowave** [reply]
For context, I'm 15 and have relationship issues. My dad has discord blocked on my computer lol. BTW I'm uploading the next update in like 2 seconds brb!!

**Reply by THEkatinamicrowave** [reply]
NEW UPDATE OUT ‼️ ‼️ ‼️

**Reply by THEkatinamicrowave** [reply]
new version is finished I'm making a tutorial as we speak

**Reply by THEkatinamicrowave** [reply]
NEW VERSION OUT ‼️ ‼️ ‼️
whoops I revamped the entire mod oopsie

**Reply by THEkatinamicrowave** [reply]
NEW UPDATE OUT

**Reply by r6915ee** [reply]
...search the Gamebanana tools section? I don't think we have any of those here in Discussions.

If you're talking about an FPS counter, press F3.

What does this have to do with someone else's script?

**Reply by r6915ee** [reply]
Then just... use that?? What are you even trying to get from piggybacking like this??
